docs/how_to_guides/miscellaneous/how_to_add_and_test_a_new_sqlalchemydataset_class.rst

```diff
@@ -0,0 +1,242 @@
+.. _how_to_guides__miscellaneous__how_to_add_and_test_a_new_sqlalchemydataset_class:
+
+How to add support for a new SQLAlchemy dialect
+===============================================
+
+This guide will help you extend the Great Expectations execution layer to work on a new SQL dialect.
+
+.. warning::
+
+   This guide is a working checklist. We improve it each time we add support for a new SQL dialect, but it still has rough edges. And adding support for a new database will probably never be a cookie-cutter operation.
+
+   If you're interested in extending Great Expectations in this way, please reach out on `Slack <greatexpectations.io/slack>`__ in the ``#contributors-contributing`` channel, and we'll work with you to get there.
+
+Steps
+-----
+
+1. Install and document dependencies
+####################################
+
+    * **Install the python dependencies for your dialect.**  While you're at it, please add a line(s) to ``requirements-dev.txt``. Examples:
+    
+        .. code-block:: bash
+        
+            pyhive[presto]>=0.6.2
+            PyAthena[SQLAlchemy]>=1.1
+            pybigquery>=0.4.15
+
+    * **Install any necessary drivers**. sqlalchemy requires special drivers for some databases. If yours is one of them, please carefully document the steps in {{document TBD}}
+    * **If possible, add docker containers for test resources.** Otherwise, document the steps to set up a suitable test environment. In either case, please document steps in {{document TBD}}.
```

Comment:
a little pointer to the CI definition here ?

Reply:
```suggestion
    * **If possible, add docker containers for test resources.** (You can find examples of similar containers in `assets/docker/`.) Otherwise, please document the steps to set up a suitable test environment.
```